src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -18,7 +18,7 @@ import {
 import EVENT_NAMES from '../../eventTracking';
 import { LEARNER_CREDIT_ROUTE } from './constants';
 import { BUDGET_STATUSES } from '../EnterpriseApp/data/constants';
-import isLmsBudget from './utils';
+import isLmsBudget from './data/utils';
```

Comment:
nit: if you're moving the `utils` under the `data` directory, it may make sense to also change `isLmsBudget` to a named export instead.

```jsx
import { isLmsBudget } from './data/utils';
```

super nit: Similarly, could also opt to introduce `./data/index.js` and export everything from `utils`.

```jsx
// ./data/index.js
export * from './utils';
```

```jsx
import { isLmsBudget } from './data';
```